### ISSUE TITLE
Identity isn't working with Netlify CMS + Hugo installation

### ISSUE THREAD

**Issue by Zerotask** [netlify/netlify-identity-widget#488]
I created a new Website with Netlify CMS + Hugo, which automatically sets everything up.
Although I can't login into the admin panel. I tried several things which are described here: https://answers.netlify.com/t/common-issue-netlify-cms-git-gateway-email-not-confirmed/10690

After creation, I updated the subdomain from the auto-generated one to 'nastyville-test'.
Creating new identity users, trying different users / emails or deleting and re-creating the identity widget didn't work.

I'd expect that this works out of the box, since it used just the Netlify tools to build it.

The URL is: http://nastyville-test.netlify.app/

**Comment by RoneoOrg** [comment]
Hi @Zerotask ,

I had a similar issue with Hugo + NetlifyCMS and finally fixed it.
The code [is public](https://github.com/iaeiou/wonderful-cms-on-netlify), and the result [can be seen there](https://wonderful-hugo-cms.netlify.app/)

Hope you'll find relevant bits.

(Note that your site is 404. If your source code is public too, you may want to post a link)

**Comment by mmm8955405** [comment]
I don't understand, just need online editing and preview function!! However, the CMS must have a GitHub account or a platform account hosted by other code. I didn't explain it. I don't know how many people have encountered this problem. I can only say garbage design

**Comment by RoneoOrg** [comment]
> However, the CMS must have a GitHub account or a platform account hosted

Yes, you need to host your files somewhere (GitHub or Gitlab) and setup Netlify Identity.

This obviously makes the first steps pretty hard, but you can find support on the [Hugo](https://discourse.gohugo.io/) and [Netlify forums](https://answers.netlify.com).

The community is welcoming and active, I hope you'll get relevant answers quickly
